pages/content/amp-dev/about/page-experience.html

```diff
@@ -3,6 +3,95 @@
+              <div class="ap-pie-charts-chart-part"></div>
+            </div>
+            <div class="ap-pie-charts-chart-clip">
+              <div class="ap-pie-charts-chart-part"></div>
+            </div>
+            <div class="ap-pie-charts-chart-overlay">
+              <span>12%</span>
+              <aside>non-AMP<br>domains</aside>
+            </div>
+          </div>
+
+          <figcaption>Domains built with AMP are 5 times more likely to pass* Core Web Vitals metrics than non-AMP!<br><span>*where „passing“ means 75% of pages in the domain passed</span></figcaption>
+        </figure>
+      </div>
+      <div class="ap-text-media-col">
+        <div class="ap-text-block">
+          <h3>AMP is a straightforward way to a high page experience score</h3>
+          <p>AMP is flexible and customizable while still enforcing restrictions that provide a reliable user experience. This makes it easy to build feature-rich pages without sacrificing performance. AMP also comes with tools to test your site and apply optimizations.</p>
+        </div>
+        <div class="ap-text-block">
+          <h3>Measure AMP page performance</h3>
+          <p>Ensure your AMP pages are exceeding your page experience goals with the <a href="https://amp.dev/page-experience/?_gl=1*8x4tg9*_ga*YW1wLUdKZEw3Q1dzRVZVTWxFZTYwU2FwR0E.">AMP Page Experience Guide</a>. This tool provides AMP-specific feedback to improve page experience metrics.</p>
+          <p>If your AMP page fails and the tool provides no feedback, it will prompt you to file an issue. A dedicated AMP team member will follow up for personal support to address any problems.</p>
+          <p>Read <a href="https://blog.amp.dev/2020/10/13/meet-amps-page-experience-guide/">Meet AMP&rsquo;s Page Experience Guide blog post</a> to learn more about the tool!</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="ap-section --fullscreen">
+    {% do doc.styles.addCssFile('css/components/atoms/pixi-stripes.css') %}
+    {% do doc.icons.useIcon('icons/pixi-stripes.svg') %}
+    <aside class="ap-a-pixi-stripes --block"></aside>
+  </section>
+
+  <section class="ap-section --col-4-23">
+    <div class="ap-text-media --thirds">
+      <div class="ap-text-media-col">
+        <div class="ap-text-block">
+          <h2>{{ _('Get started with AMP') }}</h2>
+          <p>AMP is open-source! This means it&rsquo;s already supported on many CMS&rsquo;, such as <a href="https://amp-wp.org/">WordPress,</a> and integrated with framework favorites like <a href="https://blog.amp.dev/2020/07/28/introducing-the-eleventy-amp-plugin/">Eleventy</a> and <a href="https://nextjs.org/docs/api-reference/next/amp">Next.js</a>.<br><br>If you&rsquo;re working with a custom solution, use <a href="https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/amp-optimizer-guide/?format=websites">AMP optimizers</a> on your build-system, server, or hosting environment.</p>
```

Comment:
The `&rsquo;` here should probably simply be `'` - also line breaks would help code readability.